Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This file is part of dirtsand.
+#
+# dirtsand is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# dirtsand is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with dirtsand.  If not, see <http://www.gnu.org/licenses/>.
+
+FROM alpine
+
+COPY . /usr/src/dirtsand
+WORKDIR /usr/src
+
+ARG PRODUCT_BRANCH_ID=1
+ARG PRODUCT_BUILD_ID=918
+ARG PRODUCT_BUILD_TYPE=50
+ARG PRODUCT_UUID=ea489821-6c35-4bd0-9dae-bb17c585e680
+ARG DS_HOOD_USER_NAME=DS
+ARG DS_HOOD_INST_NAME=Neighborhood
+ARG DS_HOOD_POP_THRESHOLD=20
+
+# Everything coalesced into this single RUN statement to avoid creating WIP images.
```

Comment:
I realize this can save build host resources and/or a performance enhancement that comes with having fewer filesystem layers, but it kills on time if a build fails and make debugging build stages harder.

Best practice would be to create a flat-er image at the end would be to utilize a multi-stage build process.
https://docs.docker.com/develop/develop-images/multistage-build/